### ISSUE TITLE
Chat Moderation: Use Websocket to get messages

### ISSUE THREAD

**Issue by jeyemwey** [owncast/owncast#569]
As far as I can tell, the `/admin/chat/` page loads the `/api/admin/chat/messages` only once. If you want to respond to a new message, you'd need to reload the admin panel or at least this component.

There are two ways to resolve this:

1. A `setInterval` that requests `/api/admin/chat/messages` every `n` seconds and updates the table properties
2. A WebSocket similiarly to the Chat WebSocket on the viewing page. Probably overkill?

**Comment by gabek** [comment]
If we were to add this we'd want to use the websocket, and that's not something we have planned to put in right now, mostly because should also do #537 to make it work.

I know it's not perfect, but we needed the bare minimum, and it'll be iterated on over time.  There's a _ton_ of admin work that's getting done, and everything will continue to get iterated on.  But I don't think we want to hold up getting a base version out to add real-time updates to it.

**Comment by jeyemwey** [comment]
Okay, I'll keep this one open so we know it has to be done at a future time.

This is a backlog issue for after #537 is done and we have time for that stuff.

**Comment by gabek** [comment]
Closing this in favor of https://github.com/owncast/owncast/issues/841, a new admin dashboard that includes real-time chat and moderation.
